modules/creating-a-machine-pool-cli.adoc

```diff
@@ -66,6 +67,7 @@ For fault-tolerant worker machine pools, choosing a Multi-AZ machine pool distri
 * A Single-AZ machine pool with one availability zone can have a machine count in multiples of 1, such as 1,2,3,4 and so on.
 ====
 <10> Optional: For machine pools in clusters that do not have Red Hat managed VPCs, you can select additional custom security groups to use in your machine pools. You must have already created the security groups and associated them with the VPC that you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for security groups in the "Additional resources" section.
+<11> Optional: For BYO VPC clusters, you can select a subnet to create a Single-AZ machine pool.
```

Comment:
Let's clarify that 
If the subnet is out of cluster creation subnets, there must be a tag "kubernetes.io/cluster/<cluster infra ID>: shared". it is for classic only.
And here is way how to get the cluster infra ID: 
```
lixue@Xue-Lis-MacBook-Pro rosa % rosa describe cluster -c 29rdopsh1f7e2b53gr00o2jlk74cc3pd|grep "Infra ID:" 
Infra ID:                   sdq-ci-weaxr-n8qff
lixue@Xue-Lis-MacBook-Pro rosa %
```

Reply:
@oriAdler can you verify please thanks

Reply:
Yes, that's accurate, we have this check in the backend:
checks that the subnet has the tag `kubernetes.io/cluster/<infra-id> : shared`
But I'm not sure if the infra-id is exposed to the user.
At this stage, the preflight fails, and afterward, the user adds the tag and re-attempts.

Reply:
@oriAdler Actually this output comes from rosa describe cluster. I mean the `Infra ID` is exposed for classic clusters. Do you mean there is any capability or toggle to control the cluster description?

Reply:
My bad, I missed that, thanks for the clarification, if that is the case we can add the comment about the subnet's tag.